garrysmod/lua/includes/extensions/util.lua

```diff
@@ -195,7 +198,7 @@ local T =
 	--
 	Started = function( self )
 
-		return self.endtime != nil
+		return self.endtime != nil and self.m_timestamp > 0

```

Comment:
I don't think this change is necessary at all?

Reply:
Ok, I will undo that change.